tests/unit/test_local_session.py

```diff
@@ -156,6 +156,19 @@ def test_create_model(LocalSession):
     assert 'my-model' in sagemaker.local.local_session.LocalSagemakerClient._models
 
 
+@patch('sagemaker.local.local_session.LocalSession')
+def test_delete_model(LocalSession):
+    local_sagemaker_client = sagemaker.local.local_session.LocalSagemakerClient()
+    model_name = 'my-model'
```

Comment:
You don't have to make this change, however I believe some tests are sharing common variables, such as this one. I think it would great if we could declare these near the top, so that they can be shared. :D